### PR TITLE
devel/hs-haskell-language-server: Update to 1.4.0

### PR DESCRIPTION
**Pull request by alexDarcy**
Ghc-api-compat is now managed as a "normal" dependency.
Added tasty & co. as dependencies.

### CODE REVIEW

File: devel/hs-haskell-language-server/Makefile [hunk 1]
```diff
@@ -224,7 +232,6 @@ PLIST_FILES=		bin/${PORTNAME} \
 			bin/${PORTNAME}-wrapper
 
 post-extract:
-	${MV} ${WRKDIR}/ghc-api-compat-8fee87eac97a538dbe81ff1ab18cff10f2f9fa15 ${WRKSRC}/ghc-api-compat
 	${MV} ${WRKSRC}/stack-8.10.5.yaml ${WRKSRC}/stack-8.10.7.yaml
```
Comment (arrowd) [inline devel/hs-haskell-language-server/Makefile:235]: This line isn't needed anymore too.

**Review by arrowd — CHANGES_REQUESTED** [review]
Can you please change the commit message to just "devel/hs-haskell-language-server: Update to 1.4.0" ?

**Comment by alexDarcy** [comment]
I've applied your suggestions, thanks.

**Comment by arrowd** [comment]
Thanks for update for this complex port!

**Pull request merged**